test/test_math_ops.cpp

```diff
@@ -468,6 +468,33 @@ TEST(MathOps, Log2d) {
   opexecuter.RunTest();
 }  // end of test op Log
 
+// Test op: Max
+TEST(MathOps, Max) {
+  int dim1 = 2;
+  int dim2 = 3;
+
+  Tensor A(DT_FLOAT, TensorShape({dim1, dim2}));
+  AssignInputValuesRandom(A);
+
+  // axis at which the dimension will be inserted
+  // should be -rank-1 <= axis <= rank
+  vector<int> axis_ = {0,-1};
+  
+  vector<int> static_input_indexes = {1};
+  vector<DataType> output_datatypes = {DT_FLOAT};
+  
+  for(auto const& axis:axis_){
```

Comment:
We decided to have a separate test for each case for unit tests. Please add separate tests for axis 0 and -1.

Reply:
All other tests with a similar inputs(axis to be specific) are done in this manner.

Reply:
That's fine. We decided to use this approach for the new tests that we implement.